core/src/main/java/org/lflang/diagram/synthesis/styles/LinguaFrancaStyleExtensions.java

```diff
@@ -86,6 +94,11 @@ public KRendering underlineSelectionStyle(KRendering r) {
 
   public KRendering boldLineSelectionStyle(KRendering r) {
     float lineWidthValue = _kRenderingExtensions.getLineWidthValue(r);
+    // Improve this with content from https://github.com/lf-lang/rfcs/pull/3
+    boolean selectionColor = getBooleanValue(SELECTION_HIGHLIGHTING_COLOR);
+    if (selectionColor) {
+      _kRenderingExtensions.setSelectionForeground(r, Colors.ORANGE_1);
+    }
```

Comment:
Simply add the selection foreground rendering to renderings if the synthesis option is enabled.